src/main/scala/firrtl/transforms/ConstantPropagation.scala

```diff
@@ -101,7 +101,7 @@ object ConstantPropagation {
 
 }
 
-class ConstantPropagation extends Transform with DependencyAPIMigration {
+class ConstantPropagation extends Transform with RegisteredTransform with DependencyAPIMigration {
```

Comment:
Can you check if this works? Pass a `--help` to `FirrtlStage` and make sure this shows up. I think if this goes the `RegisteredTransform` route then it needs to add to https://github.com/chipsalliance/firrtl/blob/master/src/main/resources/META-INF/services/firrtl.options.RegisteredTransform. The linkage is annoying, but there's no other way to make something discoverable like this unless you (1) are looking for things that mix-in a sealed trait or (2) very slowly walk all the classes.

Reply:
Sorry, I didn't check it actually, I have fixed that in up-to-date commits.